stories/Stepper/NavigationSteps.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { StepperContainer, StepperHeader, StepperHeaderElement, Button, Icon } from '../../src'
+import StepperContent from '../../src/components/Stepper/StepperContent'
+import StepperNav from '../../src/components/Stepper/StepperNav'
+
+const NavigationOfTheSteps = () => {
+  return(
+    <StepperContainer>
+      <StepperHeader>
+        <StepperHeaderElement variant="confirmed" icon="it-check">
+          Label Step 1
+        </StepperHeaderElement>
+        <StepperHeaderElement variant="active">
+          Label Step 2
+        </StepperHeaderElement>
+        <StepperHeaderElement>Label Step 3</StepperHeaderElement>
+        <StepperHeaderElement variant="steppers-index">
+          2/6
+        </StepperHeaderElement>
+      </StepperHeader>
+      <StepperContent>
+        Example content of the current step
+      </StepperContent>
+      <StepperNav>
+        <Button 
+          outline
+          color="primary"
+          size="sm"
+          className="steppers-btn-prev">
+            <Icon icon="it-chevron-left" color="primary"/>
+            Back
+        </Button>{' '}
+        <Button
+          outline
+          color="primary"
+          size="sm"
+          className="steppers-btn-next">
+            Next
+            <Icon icon="it-chevron-right" color="primary"/>
+        </Button>
+        <Button 
+          color="primary"
+          size="sm"
+          className="stepper-btn-confirm"
+          >
+            Confirmation
+        </Button>
```

Comment:
@dej611  please check this part of the code. Buttons don't render as they should. Is something wrong with my syntax?

Reply:
Just a typo in the `StepperNav` component which was preventing the CSS to kick-in to model all its content. :)

Reply:
Oops ! Sorry. This is so silly ! Thanks 😃